### ISSUE TITLE
the folder structure is way too complex

### ISSUE THREAD

**Issue by guzhaoyuan** [thu-skyworks/Roboseum#8]
The problem I encounter is that in the software folder, there are too many sub directories. I understand that software is a large and sophisticated  system in Robotics. But it is really hard to find what I want to read in such a file structure. I suggest either simplify the structure, or make a clear xmind picture in the readme page to help guide through the museum.

**Comment by Yiping-Steven** [comment]
Thanks for the suggestion. I'm aware of this problem. I've already added an Xmind picture in the readme page on Dec 31, 2018. Did you miss that or you think it's not clear enough? 

My current solution to this problem is avoiding browsing the file folders directly, but navigating in the Xmind and opening the markdown file in the Xmind directly. I attached a relative link to every node in the Xmind tree. I think this solution is better than simplifying the structure.( I think the sturcture should reflect the complexity and diversity of algorithms)  And I suggest hardware tree can also adopt this strategy (add an external relative link to all the markdown files in the Xmind.)
